x-pack/plugin/esql/compute/build.gradle

```diff
@@ -36,32 +36,40 @@ spotless {
   }
 }
 
-def prop(Type, type, Wrapper, TYPE, BYTES, Array, Hash) {
+def prop(Name, Type, type, Wrapper, TYPE, BYTES, Array, Hash) {
   return [
+    // Name of the DataType. Use in DataType names
+    "Name" : Name,
```

Comment:
Everything is the same for the other types, but now we have the "Name" as the specific datatype. It's a bit messy, tried to describe the things a bit. If we need to add more types later (Geo? Version?), we can rethink it a bit.
But for this one, just adding the name was enough

Reply:
I always thought of `Type` as the name and `type` as the name of the elements. I feel like we may want to do a pass after this to try and rename some of these variables. Maybe reduce the usages of the boolean returning ones. It's grown a bit out of hand.